js/framework/application.js

```diff
@@ -253,15 +254,25 @@ var Application = Class.inherit({
     _disposeRemovedViews: function() {
         var that = this,
             args;
+
+        var disposeView = function(viewInfo) {
```

Comment:
I see no reason to recreate this function on each method call. It can be a regular class (private) method.